cpp/oneapi/dal/algo/pca/backend/cpu/train_kernel_cov.cpp

```diff
@@ -67,7 +65,7 @@ static train_result<Task> call_daal_kernel(const context_cpu& ctx,
 
     constexpr bool is_correlation = false;
     constexpr uint64_t results_to_compute =
-        int64_t(daal_pca::mean | daal_pca::variance | daal_pca::eigenvalue);
+        std::int64_t(daal_pca::mean | daal_pca::variance | daal_pca::eigenvalue);
```

Comment:
Should we explicitly declare type of this variable?

Reply:
fixed